client/reducers/index.js

```diff
@@ -66,6 +66,20 @@ export const getMessagesByStory = (state:Object, storyUid:string) => {
   return R.pickBy(isFromStory, getMessages(state))
 }
 
+export const byDateDescending = R.comparator((d1, d2) => d1.sent > d2.sent)
+export const getGroupMessages = (state, {params}) => {
```

Comment:
If the only argument we're passing is groupUid, then let's make that the second argument rather than `{params}`, then we'll just return the messages instead of an object with a messages key.

I noticed that this is being sued in place of the stateToProps function entirely, that makes it pretty tough to extend. Let's change the parameters then write a custom stateToProps each place it's used.